content/hardware/01.mkr/01.boards/mkr-nb-1500/tutorials/nb-scan-network/nb-scan-network.md

```diff
@@ -43,7 +43,7 @@ The goals of this project are:
 
 ## The uBlox SARA-R4 Module
 
-As every other MKR family board, the MKR NB 1500 board has a specific module for connectivity. It is called uBlox SARA-R4, and is designed to communicate over LTE Cat M1 or NB IoT networks with a speed of up to 375 kb/s.
+As every other MKR family board, the MKR NB 1500 board has a specific module for connectivity. It is called uBlox SARA-R4, and is designed to communicate over LTE Cat M1 or NB IoT networks with a speed of up to 375 KBps.
```

Comment:
Should it be Bytes or Bits? 🤔

Reply:
According to the Datasheet which is published on [content.arduino.cc](https://content.arduino.cc/assets/Arduino_SARA-R4_DataSheet_%28UBX-16024152%29.pdf#page=8) the data rate of "LTE category M1" is given in kb/s (kilobytes per second). This datasheet has the revision 18. On the internet I found a newer version (R-29) of the [datasheet](https://content.u-blox.com/sites/default/files/SARA-R4_DataSheet_UBX-16024152.pdf#page=12). There are some data rates which are given in kb/s and some are given in kbits/s (kilobits per second). But the interesting one (LTE category M1) is given in kb/s. So I think that bytes are correct.
If you have another opinion or another source please write a comment :)

Reply:
In the datasheet that you've shared it says 375 kb/s (kilo bits) so I assume your change is not needed.

![image](https://user-images.githubusercontent.com/11246294/182131646-19c68acf-86e7-46b4-a854-190d0e140c1b.png)

Also if the datasheet that we've shared is not up to date I will update that add it 👍

Reply:
The data sheet (revision 18) mentions 375kb/s, the newer data sheet (revision 29) also 375kb/s. But the point is that in the column to the right of the 375kb/s (in the new datasheet) there is an indication in kbit/s. So why does the datasheet say kbit/s and kb/s when it should mean the same thing. But if kilobit is correct, then I need to change kb/s to kbps to satisfy the linter rule.
FYI: The image you have shared is from the old data sheet.

Reply:
Screenshot from https://content.u-blox.com/sites/default/files/SARA-R4_DataSheet_UBX-16024152.pdf (R2)
Yes please change it to kbps and lets merge it! 🙌

Reply:
```suggestion
As every other MKR family board, the MKR NB 1500 board has a specific module for connectivity. It is called uBlox SARA-R4, and is designed to communicate over LTE Cat M1 or NB IoT networks with a speed of up to 375 kbps.
```